libraries/AC_CustomControl/AC_CustomControl.cpp

```diff
@@ -193,4 +193,4 @@ void AC_CustomControl::set_notch_sample_rate(float sample_rate)
 #endif
 }
 
-#endif
+#endif
```

Comment:
Can you revert the removal of trailing lines in AC_CustomControl?

Reply:
Yeah, I'll do it